docs/compilation.rst

```diff
@@ -458,33 +460,6 @@ From the root of the DALI source tree
 
 The relevant python wheel will be in ``dali_root_dir/wheelhouse``
 
-Cross-compiling for aarch64 QNX (Docker)
-----------------------------------------
-
-.. note::
-
-  Support for aarch64 QNX platform is experimental. Some of the features are available only for
-  x86-64 target and they are turned off in this build. There is no support for DALI Python library
-  on aarch64 yet. Some Operators may not work as intended due to x86-64 specific implementations.
-
-Setup
-^^^^^
-After aquiring the QNX Toolchain, place it in a directory called ``qnx`` in the root of the DALI tree.
-Then using the SDK Manager for NVIDIA DRIVE, select **QNX** as the *Target Operating System*
-and select **DRIVE OS 5.1.0.0 SDK**.
-
-In STEP 02 under **Download & Install Options**, select *Download Now. Install Later*.
-and agree to the Terms and Conditions. Once downloaded move the **cuda-repo-cross-qnx**
-debian package into the ``qnx`` directory you created in the DALI tree.
-
-Build the aarch64 Build Container
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: bash
-
-    docker build -t nvidia/dali:tools_aarch64-qnx -f docker/Dockerfile.cuda_qnx.deps .
-    docker build -t nvidia/dali:builder_aarch64-qnx --build-arg "QNX_CUDA_TOOL_IMAGE_NAME=nvidia/dali:tools_aarch64-qnx" -f docker/Dockerfile.build.aarch64-qnx .
-
```

Comment:
The "Compile" section below still has QNX instructions. Is that intended?

Reply:
Not really. How about now?